release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -86,6 +86,25 @@ The {product-title} installation program for Microsoft Azure now creates subnets
 [id="ocp-4-9-security"]
 === Security and compliance
 
+[id="ocp-4-9-security-custom-audit-policy"]
+==== Configuring the audit log policy with custom rules
+
+You now have more fine-grained control over the audit logging level for {product-title}. You can use custom rules to specify a different audit policy profile (`Default`, `WriteRequestBodies`, `AllRequestBodies`, or `None`) for different groups.
+
+For more information, see xref:../security/audit-log-policy-config.adoc#configuring-audit-policy-custom_audit-log-policy-config[Configuring the audit log policy with custom rules].
```

Comment:
@bergerhoffer Just reminder, the current link is pointing incorrectly, but I suspect that will change later.

Reply:
@wangke19 Yes, sorry I noted this in the first comment of the PR, but should have also mentioned it directly to you. I will wait to merge this PR until #36229 merges.